userspace/libsinsp/examples/test.cpp

```diff
@@ -25,9 +25,29 @@ limitations under the License.
 
```

Comment:
@Molter73 let me know if with this change I have broken your tests, if you use only the kernel module you shouldn't notice the difference. Let me know if you have some troubles with it :(